src/components/contact_us/components/send_mail_form/sendForm.less

```diff
@@ -70,8 +70,12 @@
 	color: var(--white-color)
 }
 .not_valid {
-	border-color: red !important;
+	border-color: var(--not-valid-color) !important;
 }
 .not_valid_label {
```

Comment:
If you set a label as an html `label` container tag - the html code will be simpler and: 
* Clicking on the label automatically focus the underlying input - that's good for us
* All the forms on the public website must be 100% accessible - `label > input` sets it right